src/capsule/react-native/ReactNativeChallengeStorage.ts

```diff
@@ -8,15 +12,30 @@ const PEM_HEADER = '-----BEGIN PUBLIC KEY-----'
 const PEM_FOOTER = '-----END PUBLIC KEY-----'
 
 export class ReactNativeChallengeStorage extends ChallengeStorage {
+  protected signOptions(): BiometryParams {
+    return {
+      biometryTitle: 'Authenticate',
+      biometrySubTitle: 'Signing',
+      biometryDescription: 'Authenticate yourself to sign the text',
```

Comment:
Does this get shown to the user? If so we may want to run by the team to get consensus on what we want this to say. Also long term we'll prob want to localize it.

Reply:
user can personalize it, but yep, it's visible